src/tasks/buildWithCompose.ts

```diff
@@ -27,7 +29,7 @@ export function buildWithCompose({
       title: "Build docker image",
       task: async (_, task) => {
         // Prior to this task, the compose should had been updated with the proper tag
-        await shell(`docker-compose --file ${composePath} build`, {
+        await shell(`docker-compose --file ${composePath} ${variantComposePath ? `--file ${variantComposePath} ` : ""}build`, {
```

Comment:
same as above.

Also is really weird to have an empty string as a value of the flag